File: PRAKTIKUM_4/5/PRAK405-2410817110010-DanielNoprianto.c
#include <stdio.h>
int main (){
    int i, j;
    int baris, pengali, total=0;

    scanf("%d %d", &baris, &pengali);

    for (i = 1; i <= baris; i++){
        int a = 0;
        printf("(");
        
        for (j = 1; j <= i; j++){
            printf("%d * %d", j, pengali);
            a += j * pengali;

            if (i>j){
                printf(") + (");
            }

            else if (j>i){
                printf(") + (");
            }   
        }
        printf(") = %d\n", a);
        total += a;
    }
    printf ("%d\n", total);
    return 0;
}

File: PRAKTIKUM_4/5/PRAK405-2410817110010-DanielNoprianto.py
baris, pengali = map(int,input("").split())
total = 0

for i in range (1, baris + 1):
    a = 0
    print("(", end="")

    for j in range (1, i + 1):
        print(f"{j} * {pengali}", end="")
        a += j * pengali

        if j < i or i > j:
            print(") + (", end="")

    print(f") = {a}")
    total += a

print(f"{total}")